### PR TITLE
update(docs): Update docs with latest experiment tunables

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/pod-memory-hog.md [hunk 1]
```diff
@@ -141,7 +141,19 @@ subjects:
...
-  </tr>      
+  </tr>
+  <tr>
+    <td> TARGET_POD </td>
+    <td> Name of the application pod subjected to pod memory hog chaos<td>
+    <td> Optional </td>
+    <td> If not provided it will select from the app label provided</td>
+  </tr>
+  <tr>
+    <td> CHAOS_KILL_COMMAND </td>
```
Comment (ksatchit) [inline docs/pod-memory-hog.md:152]: CHAOS_INJECT_COMMAND also here?

**Comment by ksatchit** [comment]
Can we place this on versioned-docs also?

File: docs/pod-memory-hog.md (hunk 1 above)
Comment (uditgaurav) [reply docs/pod-memory-hog.md:152]: Not in env https://github.com/litmuschaos/litmus-go/blob/master/pkg/generic/pod-memory-hog/environment/environment.go

File: website/versioned_docs/version-1.8.0/litmus-demo.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+---
+id: litmus-demo
+title: Chaos Engineering in a Microservices Environment
+sidebar_label: Litmus Demo
```
Comment (ksatchit) [inline website/versioned_docs/version-1.8.0/litmus-demo.md:4]: versioned docs have a diff metadata fields

Comment (uditgaurav) [reply website/versioned_docs/version-1.8.0/litmus-demo.md:4]: ok, Thanks!

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**